packages/core/helper-plugin/src/components/Form.tsx

```diff
@@ -2,23 +2,23 @@ import React, { useEffect, useRef } from 'react';
 
 import { Form as FormikForm, useFormikContext } from 'formik';
 
-const Form = (props) => {
-  const formRef = useRef(null);
+const Form = ({ ...props }) => {
```

Comment:
@markkaylor do we use this anywhere? It's really bad code imo, i'm wondering if it's best we deprecate this and remove it's usage – what do you think?

Reply:
i think deprecate first cause then everyone will see it and can action it as they go 🤓

Reply:
Sounds good to me. @Tegacreatives could you add an `@deprecated` annotation with a message to use Formik Form directly instead.

Reply:
Sure @markkaylor